### DISCUSSION TITLE
flutter Supabase magic link/redirect link not working on emulator

### DISCUSSION THREAD

**Discussion by ujaiahmad** [supabase/supabase#9870] [Questions] [answered]
Hi,

I'm trying to follow the [flutter quickstart](https://supabase.com/docs/guides/with-flutter) example. Everything seems to be working fine for the authentication when I run it on a browser like Chrome or Edge. But when I run it on an android emulator, it runs fine until I click on the magic link from my email. It won't redirect to the application in the emulator. I also tried this on my device but still the same problem. 
<img src=https://user-images.githubusercontent.com/25504609/198094323-86efa4cd-a811-4293-b4bc-bd5f4f0d1919.jpg width=350 height=600/>
<img src="https://user-images.githubusercontent.com/25504609/198094338-571c9c57-4a28-4d63-8ff6-3c6d7e015a5e.jpg" width=350 height=600/>
Anyone can help me please, thanks.

**Comment by dshukertjr** [comment, marked as answer]
@ujaiahmad There was an issue that prevented this feature from working properly. It should be working now. Could you delete your pubspec.lock file and rerun `pub get` to get the newest version of gotrue-dart and retry it?

**Reply by ujaiahmad** [reply]
Yes, I did that and change the redirect to ''io.supabase.flutter://SignUp-callback/'' instead of local host and now it runs properly. Thank you
